### PR TITLE
Fixes and updates

### PR DESCRIPTION
**Pull request by rozyczko**
This now generates a properly rendered project, passing tests.

### CODE REVIEW

**Comment by rozyczko** [comment]
this now works as expected generating a template project which passes the build/testing stage.

**Comment by andped10** [comment]
Are you able add a link for a project that has been created from the template?

File: template/pyproject.toml.jinja [hunk 1]
```diff
@@ -74,7 +77,7 @@ source = ["src/{{packagename}}"]
 
 [tool.github.info]
 organization = 'easyScience'
-repo = {{packagename}}Lib
+repo = "{{packagename}}"
```
Comment (andped10) [inline template/pyproject.toml.jinja:80]: Should we make it possible to handle the `Lib` cases by adding a `repo_postfix` variable

File: README.md [hunk 2]
```diff
@@ -3,11 +3,53 @@
 See [copier](https://copier.readthedocs.io/en/stable/) for details.
```
Comment (andped10) [inline README.md:3]: Maybe add a few lines on how to setup `pre-commit`

File: template/pyproject.toml.jinja [hunk 3]
```diff
@@ -63,9 +60,15 @@ docs = [
 homepage = "https://docs.{{packagename}}.org"
 documentation = "https://docs.{{packagename}}.org"
 
+[tool.hatch.version]
```
Comment (andped10) [inline template/pyproject.toml.jinja:63]: It was attempted to get the version nr from the git tag using:
`setuptools-git-versioning`

However I have not yet been able to get that functionality to work.

**Comment by rozyczko** [comment]
> Are you able add a link for a project that has been created from the template?

Sure.
The actions are here:
https://github.com/rozyczko/EasyUtilities/actions/runs/8449487898

The project is here:
https://github.com/rozyczko/EasyUtilities

File: template/src/{{packagename}}/__version__.py.jinja [hunk 4]
```diff
@@ -0,0 +1 @@
+__version__ = "0.0.0"
```
Comment (andped10) [inline template/src/{{packagename}}/__version__.py.jinja:1]: It was attempted to get the version from `importlib.metadata.version` in `__init__.py.jinja`.

But let's stick to what is suggested here unless you would like to play around and get the other way working.

File: template/pyproject.toml.jinja (hunk 3 above)
Comment (andped10) [reply template/pyproject.toml.jinja:63]: Also, see comment below

**Review by andped10 — APPROVED** [review]
Looks good to me

**Comment by andped10** [comment]
In the created project I noted a lot of warnings for: `codecov/codecov-action@v3` and I suggest to bump to `v4`

**Comment by andped10** [comment]
I made a change to the `README.md.jinja` file.  However, I don't think that is related to the failing pipeline

**Comment by andped10** [comment]
We might also want to add the pipelines for (in ERL development branch):
- `release-drafter.yml`
- `workflows/release-drafter.yml`
- `workflows/release-drafter-verify-pr-labels.yml`

File: template/pyproject.toml.jinja (hunk 1 above)
Comment (rozyczko) [reply template/pyproject.toml.jinja:80]: On the other hand, we might want this to be more generic. On the one hand, "EasyQens" will have both "Lib" and "App" but other projects like "EasyUtilities" won't have a `repo_prefix`, so maybe specifying the name directly is more sensible?

**Comment by rozyczko** [comment]
> I made a change to the `README.md.jinja` file. However, I don't think that is related to the failing pipeline

Tests were failing due to breaking changes in the new version of `pyyaml-include` package.
Reverted to 1.x and things work now.

**Pull request merged**